psalm.baseline

```diff
@@ -35,6 +35,48 @@
       <code>scalarNode</code>
     </PossiblyUndefinedMethod>
   </file>
+  <file src="src/Service/CognitoIdentityProvider/src/Result/AdminCreateUserResponse.php">
```

Comment:
hmmm.. I'm not a huge fan of ignoring ~ everything

Reply:
Lets fix that before 0.1.0